core/src/com/unciv/ui/utils/ResponsePopup.kt

```diff
@@ -0,0 +1,17 @@
+package com.unciv.ui.utils
+
+import kotlin.concurrent.thread
+
+class ResponsePopup (message: String, screen: CameraStageBaseScreen, time: Long = 1000) : Popup(screen){
```

Comment:
I'd really like to see a picture of this, it sounds like something we really needed in the game :)

Reply:
Here is a picture of it right after starting a new multiplayer game. It stays for 2.5 seconds
![r_popUp_after_Game_Start](https://user-images.githubusercontent.com/24532072/74652144-ae00a700-5185-11ea-9ca6-c59f10b2b031.png)